Doc/c-api/dict.rst

```diff
@@ -123,7 +123,7 @@ Dictionary Objects
 
    This is the same as the Python-level :meth:`dict.setdefault`.  If present, it
    returns the value corresponding to *key* from the dictionary *p*.  If the key
-   is not in the dict, it is inserted with value *defaultobj* and *defaultobj*
+   is not in the dict, it is inserted with value *default* and *default*
```

Comment:
Actually, the signature in `Objects\dictobject.c` is
```
PyObject *
PyDict_SetDefault(PyObject *d, PyObject *key, PyObject *defaultobj)
```

so I believe this needs to change in the function definition.

Reply:
**default** is a keyword in C and can not be used as a parameter name.